README.md

```diff
@@ -10,20 +10,23 @@ Welcome to **CyberBear**, a gamified platform designed to educate users on cyber
   - [Technology Stack](#technology-stack)
   - [Use Cases](#use-cases)
   - [Setup and Installation](#setup-and-installation)
-  - [Contribution Guidelines](#contribution-guidelines)
-    - [Development Requirements](#development-requirements)
+  - [Join Our Community](#join-our-community)
   - [License](#license)
 
 ## Project Overview
 CyberBear is part of **HackToFuture-2024**, under the Cyber Security domain (Problem Statement Code: CS-01). Our team aims to promote cybersecurity awareness by turning complex cybersecurity concepts into interactive games, utilizing a subscription model to unlock advanced features, and providing quizzes and real-time assistance with the help of a fine-tuned Large Language Model (LLM).
 
+![Gamified Learning](https://media.giphy.com/media/3o7abB06u9bNzA8lu8/giphy.gif)
+
 ## Features
 - **Gamified Learning**: Fun, interactive games to teach cybersecurity concepts.
 - **Subscription Model**: Additional tools and resources for subscribed users.
 - **Quizzes with Feedback**: Test knowledge with quizzes and get instant feedback.
 - **Chatbot Assistant**: 24/7 assistance via a fine-tuned LLM model.
 - **Tools & Resources**: Curated cybersecurity resources for individuals and businesses.
 
+![Quizzes with Feedback](https://media.giphy.com/media/l0HlSNOxJB956qwfK/giphy.gif)
+
```

Comment:
I don't understand how these gifs are related to out work?

Reply:
So, should I remove all the gifs or should I add relevant gifs?

Reply:
Add the relevant ones